examples/mf6/circle_transport.py

```diff
@@ -250,9 +250,9 @@
 transport_model["adv"] = imod.mf6.AdvectionUpstream()
 transport_model["mst"] = imod.mf6.MobileStorageTransfer(porosity)
 
-# %%
-# Now we define initial conditions (0.0) and output options for the transport
-# simulation, and assign the transport model to the simulation as well.
+# %% Define the maximum concentration as the initial conditions, also output
+# options for the transport model, and assign the transport model to the
```

Comment:
I think the sphinx converter to Jupyter blocks requires a newline after the ``# %%`` to render properly.

```suggestion
# %%
# Define the maximum concentration as the initial conditions, also output
# options for the transport model, and assign the transport model to the
```